website/README.md

```diff
@@ -40,13 +40,9 @@ The website can be run locally through node.js or [Docker](https://www.docker.co
 
 > **Note:** If you are using a text editor that uses a "safe write" save style such as **vim** or **goland**, this can cause issues with the live reload in development. If you turn off safe write, this should solve the problem. In vim, this can be done by running `:set backupcopy=yes`. In goland, search the settings for "safe write" and turn that setting off.
 
-| :warning: WARNING :warning:                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
❓ Do we still want this?

Reply:
Seems possible - looks like this got changed in a commit directly on this repo (https://github.com/hashicorp/vault/pull/13027), rather than via `web-platform-packages`.

One option might be to move this block to after `<!-- END: local-development -->`. Or we could add it into https://github.com/hashicorp/web-platform-packages/pull/152 I guess? I feel like the latter could make sense, assuming this warning will be helpful for folks 👍

Reply:
I don't mind adding it to the package on https://github.com/hashicorp/web-platform-packages/pull/152 if that would be helpful! Also since im updating all the docs repos anyways, would be good to get all the changes in. @BRKalow thoughts?

Reply:
Ah these instructions are not valid as-is w.r.t the docker flow. I think it's alright if we just remove it.

Reply:
Works for me!